client/extensions/zoninator/components/settings/zone-creator/index.jsx

```diff
@@ -1,5 +1,65 @@
-const ZoneCreator = () => {
-	return null;
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { FormSection, reduxForm } from 'redux-form';
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormButton from 'components/forms/form-button';
+import FormButtonsBar from 'components/forms/form-buttons-bar';
+import FormLabel from 'components/forms/form-label';
+import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import SectionHeader from 'components/section-header';
+
+const ZoneCreator = ( { translate } ) => {
+	return (
+		<div>
+			<form>
+				<FormSection name="newZone">
+					<SectionHeader label={ translate( 'New Zone' ) } />
+
+					<Card>
+						<p>
+							{ translate(
+								'To create a zone, enter a name (and any other info) and click "Add Zone". ' +
+								'You can then choose content items to add to the zone.'
+							) }
+						</p>
+
+						<FormFieldset>
+							<FormLabel htmlFor="zoneName">{ translate( 'Name' ) }</FormLabel>
```

Comment:
You'll need to give the `ReduxFormTextInput` and `ReduxFormTextarea` `id` attributes that match `htmlFor` in order to set the focus on the input when the label is clicked. Right now it's not doing that.